etc/config/c++.amazon.properties

```diff
@@ -1360,6 +1360,12 @@ libs.namedtype.url=https://github.com/joboccara/NamedType
 libs.namedtype.versions.trunk.version=trunk
 libs.namedtype.versions.trunk.path=/opt/compiler-explorer/libs/NamedType/include:/opt/compiler-explorer/libs/NamedType/include/NamedType
 
+libs.pipes.name=Pipes
+libs.pipes.versions=trunk
+libs.pipes.url=https://github.com/joboccara/Pipes
+libs.pipes.versions.trunk.version=trunk
+libs.pipes.versions.trunk.path=/opt/compiler-explorer/libs/Pipes/include:/opt/compiler-explorer/libs/Pipes/include/Pipes
```

Comment:
I would advise against adding include/Pipes to avoid overlap with other libraries, and to be consistent with documented way of handling with the library. (It's not documented in the readme, but the tests/ folder does mention using include "Pipes/something" e.g. https://github.com/joboccara/pipes/blob/master/tests/do_then.cpp)

Reply:
(in the libraries that we did do it, it's mostly for backwards compatibility from when paths were wrong)

Reply:
I agree with @partouf here: ensuring it's setup for `#include "pipes/things.h"` if possible :)